### DISCUSSION TITLE
DHCP里面IPV6设置为灰色，我已经在树莓派系统里面给接口分配了静态V6地址

### DISCUSSION THREAD

**Discussion by lkxlzx** [AdguardTeam/AdGuardHome#3554] [Q&A]
我在树莓派里创建了桥，然后静态分配了IPV4及IPV6地址，IPV4能正常使用，但是DHCP里面的IPV6设置项是灰色
ifconfig 查看接口，网桥已经有V6地址

**Comment by ameshkov** [comment]
分配静态V6地址以后你重启了AGH吗？

在AGH的DHCP设置你看不看V6接口？
